hmy/blockchain.go

```diff
@@ -161,18 +161,19 @@ func (hmy *Harmony) GetPreStakingBlockRewards(
 		txFee := new(big.Int).Mul(tx.GasPrice(), big.NewInt(int64(receipts[receiptIndex].GasUsed)))
 		txFees = new(big.Int).Add(txFee, txFees)
 	}
-	for _, stx := range blk.StakingTransactions() {
-		dbsTx, _, _, receiptIndex := rawdb.ReadStakingTransaction(hmy.ChainDb(), stx.Hash())
-		if dbsTx == nil {
-			return nil, fmt.Errorf("could not find receipt for tx: %v", stx.Hash().String())
-		}
-		if len(receipts) <= int(receiptIndex) {
-			return nil, fmt.Errorf("invalid receipt indext %v (>= num receipts: %v) for tx: %v",
-				receiptIndex, len(receipts), stx.Hash().String())
-		}
-		txFee := new(big.Int).Mul(stx.GasPrice(), big.NewInt(int64(receipts[receiptIndex].GasUsed)))
-		txFees = new(big.Int).Add(txFee, txFees)
-	}
+	// todo current not add StakingTransactions tx fee to leader
```

Comment:
any impact removing this part of code? why?

Reply:
already checked with RJ. the gas fee of staking transaction never reward to leader. will delete this segment then